events/wc_offmap_events.txt

```diff
@@ -934,28 +845,509 @@ narrative_event = {
 }
 character_event = {
 	id = WCOME.250
```

Comment:
Why you kept this event?

Reply:
well the old gods have to be killed somehow and firing this event by just winning the terriory in a war is kinda weird